packages/app/src/embed/components/Content/index.tsx

```diff
@@ -38,47 +40,47 @@ import { getPreviewTabs } from '@codesandbox/common/lib/templates/devtools';
 
 import { Container, Tabs, Split } from './elements';
 
-type EmbedError = {
-  column: number,
-  line: number,
-  message: string,
-  title: string,
-  payload?: Object,
-  severity: 'error' | 'warning',
-};
-
 type Props = {
-  showEditor: boolean,
-  showPreview: boolean,
-  previewWindow: string,
-  isInProjectView: boolean,
-  setProjectView: (sandboxId?: ?string, isOpen: boolean, cb: Function) => void,
-  sandbox: Sandbox,
-  currentModule: Module,
-  hideNavigation: boolean,
-  autoResize: boolean,
-  fontSize?: number,
-  initialPath: string,
-  setCurrentModule: (moduleId: string) => void,
-  useCodeMirror: boolean,
-  enableEslint: boolean,
-  editorSize: number,
-  highlightedLines: Array<number>,
-  forceRefresh: boolean,
-  expandDevTools: boolean,
-  runOnClick: boolean,
-  verticalMode: boolean,
-  tabs?: Array<string>,
+  showEditor: boolean;
+  showPreview: boolean;
+  previewWindow: string;
+  isInProjectView: boolean;
+  setProjectView: (
+    sandboxId: string | undefined,
```

Comment:
@CompuIves This can be

```suggestion
    sandboxId?: string,
```

Reply:
This changes semantics a bit, I'm not sure if it's a required prop or not, I only know it can be undefined.